Lib/fontTools/ttLib/tables/sbixGlyph.py

```diff
@@ -71,7 +71,7 @@ def decompile(self, ttFont):
                 sbixGlyphHeaderFormat, self.rawdata[:sbixGlyphHeaderFormatSize], self
             )
 
-            if self.graphicType == "dupe":
+            if self.graphicType == "dupe" or self.graphicType == "flip":
```

Comment:
maybe you could define a method `is_reference_type` that simply `return self.graphicType == "dupe" or self.graphicType == "flip"` and use everywhere

Reply:
@anthrotype Done!